### PR TITLE
Create CCC-TAC-Minutes-2022-09-22.md

### PR DESCRIPTION
**Pull request by krtaylor**
CCC TAC Meeting Minutes from the 2022-09-22 meeting

### CODE REVIEW

File: TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+   * Accenture - Giuseppe Giordano
+   * Ant Group - Hongliang Tian (Tate)
+   * Arm - Thomas Fossati + / Michael Lu
+   * Google - Cfir Cohen  / Catalin Sandu +
+   * Huawei - Zhipeng (Howard) Huang +
+   * Intel - Dan Middleton + / Simon Johnson
+   * Meta - Eric Northup + / Shankaran Gnanashanmugam
+   * Microsoft - Dave Thaler (TAC chair) +
+   * Red Hat - Lily Sturmann +  / Dimitrios Pendarakis
+" * " indicates attending
```
Comment (dcmiddle) [inline TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md:51]:
```suggestion
" + " indicates attending
```

**Review by dcmiddle — APPROVED** [review]
Looks good. Just the typo. Thanks.

File: TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Proposed: That the minutes of the September 8, 2022 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+   * Quorum was reached and the minutes were approved by the voting members of the TAC. Catalin Sandu abstained.
+
+
+**TAC Tech Talks**
+
+   * NSF Center for Distributed Confidential Computing (CDCC) - Prof. Xiaofeng Wang
+       * Mark Novak - Possible collaboration with new Governance (GRC) SIG
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md:89]: What is above line supposed to mean?   Is there an action item?

File: TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+   * MPC + TEE - Jordan Brandt
+       * MPC Alliance Privacy Summit, Oct 5 (virtual only) [https://www.mpcalliance.org/](https://www.mpcalliance.org/)
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md:91]: Wasn't there an action item or recommendation here?   I thought there was something about followup with the Veracruz project?

File: TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+**Common Terminology whitepaper working session**
+
+   * DT and MB discussed [PR 134](https://github.com/confidential-computing/governance/pull/134) and [PR 135](https://github.com/confidential-computing/governance/pull/135) - discussion ensued
+   * General consensus on PR 135, will seek approval next meeting
+   * DT discussed [PR 132](https://github.com/confidential-computing/governance/pull/132) merged with the addition of a new PR from comment to be discussed at the next meeting
+
+**Budget Review**
+
+   * DT discussed GB request to look into funding travel for speakers
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md:116]:
```suggestion
   * DT discussed Governing Board request to look into funding travel for speakers
```

File: TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md (hunk 2 above)
Comment (krtaylor) [reply TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md:89]: Mark stated that it was just a comment

File: TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md (hunk 3 above)
Comment (AlecFernandez) [reply TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md:91]: Jordan Brandt mentioned that MPC Alliance members do work on opensource projects but that the MPC Alliance does not sponsor or focus on any opensource projects. 
Dave Thaler stated that CCC does sponsor opensource projects and named Thomas Fossati and Veracruz.  Thomas said that Dereck would be the person to speak to the high degree of overlap of goals of the two projects.
Jordan said that he and Thomas should connect to join forces and that there would probably be interest on the part of MPC alliance members.
MPC alliance summit might be a good place to make connections.
Thomas has action to make introductions

**Pull request merged**